src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, AreaCalculator {
+    protected static final double ROUNDING_PRECISION = 10.0;
```

Comment:
Why do we need this rounding precision? If so, it feels like it's better to move it AreaCalculator